crates/cheatcodes/src/inspector.rs

```diff
@@ -1080,6 +1070,28 @@ impl Cheatcodes {
             None => StdRng::from_entropy(),
         })
     }
+
+    /// Returns existing or set a default `ArbitraryStorage` option.
+    /// Used by `setArbitraryStorage` cheatcode to track addresses with arbitrary storage.
+    pub fn ensure_arbitrary_storage(&mut self) -> &mut ArbitraryStorage {
```

Comment:
can just be `fn arbitrary_storage`

Reply:
this one needs to be public as it it called when `setArbitraryStorage` to set option to some, can rename to `arbitrary_storage()`

https://github.com/foundry-rs/foundry/blob/aec556840a846074521b048271983e44906ef7a8/crates/cheatcodes/src/utils.rs#L153-L160

Reply:
yes i mean rename

Reply:
ah, kk, changed in https://github.com/foundry-rs/foundry/pull/8848/commits/0490298d37f4d9f040e05890a810f29e51341363